### DISCUSSION TITLE
What is the new parameters option in 1.29.0?

### DISCUSSION THREAD

**Discussion by jazzabeanie** [microsoft/azuredatastudio#15687] [Q&A] [answered]
I see in [the release notes](https://docs.microsoft.com/en-us/sql/azure-data-studio/release-notes-azure-data-studio?view=sql-server-ver15) for Azure Data Studio 1.29.0 that there is a new item in Notebooks - "Added runs with a parameters option". Anyone know anything more about this? I was hoping that parameters functionality would have been added to SQL kernel but seems not.

**Comment by VasuBhog** [comment, marked as answer]
Hi @jazzabeanie, please find more information here on parameterization and the [Run with Parameters](https://docs.microsoft.com/en-us/sql/azure-data-studio/notebooks/run-with-parameters) action.

**Reply by jazzabeanie** [reply]
Thanks @VasuBhog, but this doesn't tell me what functionality has been added in version 1.29.0.

**Reply by VasuBhog** [reply]
Run with Parameters action was added with the 1.29.0 release, it allows for users that have a notebook with a parameters cell to quickly inject new parameters to use for the notebook all through the action described in the docs. Was there something in the docs that you have questions or concerns about?

**Reply by Charles-Gagnon** [reply]
Apologies @jazzabeanie - it looks like the [release notes for 1.29.0](https://github.com/microsoft/azuredatastudio/releases/tag/1.29.0) on the Github releases page weren't updated to include the mention of the "Run with Parameters" functionality. 

I've added that now - typically every release will list out all the new functionality so if you see something new you can check out the release notes to find out more about I!
